distribution/pom.xml

```diff
@@ -166,34 +176,170 @@
+                        <dockerFile>src/main/docker/nginx/Dockerfile</dockerFile>
+                        <contextDir>${project.basedir}</contextDir>
+                      </build>
+                    </image>
+                  </images>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <!-- run the ITs -->
+          <plugin>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>feature-launcher-maven-plugin</artifactId>
+            <version>0.1.2</version>
+            <configuration>
+              <launches>
+                <launch>
+                  <id>websight-cms-starter</id>
+                  <feature>
+                    <groupId>${project.groupId}</groupId>
+                    <artifactId>${project.artifactId}</artifactId>
+                    <version>${project.version}</version>
+                    <classifier>websight-cms-starter</classifier>
+                    <type>slingosgifeature</type>
+                  </feature>
+                  <launcherArguments>
+                    <frameworkProperties>
+                      <org.osgi.service.http.port>${cms.port}</org.osgi.service.http.port>
+                    </frameworkProperties>
+                  </launcherArguments>
+                  <environmentVariables>
+                    <MONGODB_PORT>${mongo.port}</MONGODB_PORT>
+                    <WS_ADMIN_USERNAME>${websight.admin.username}</WS_ADMIN_USERNAME>
+                    <WS_ADMIN_PASSWORD>${websight.admin.password}</WS_ADMIN_PASSWORD>
+                  </environmentVariables>
+                  <startTimeoutSeconds>${it.startTimeoutSeconds}</startTimeoutSeconds>
+                </launch>
+              </launches>
+            </configuration>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>start</goal>
+                  <goal>stop</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <version>3.0.0-M7</version>
+            <configuration>
+              <dependenciesToScan>
+                <dependency>pl.ds.websight:websight-cms-ce-tests-integration</dependency>
+              </dependenciesToScan>
+              <systemPropertyVariables>
+                <WS_ADMIN_USERNAME>${websight.admin.username}</WS_ADMIN_USERNAME>
+                <WS_ADMIN_PASSWORD>${websight.admin.password}</WS_ADMIN_PASSWORD>
+                <WS_APPLICATION_VERSION>${websight.cms.version}</WS_APPLICATION_VERSION>
+                <INSTALLED_PACKAGES_COUNT_STARTER>1</INSTALLED_PACKAGES_COUNT_STARTER>
+                <INSTALLED_PACKAGES_GROUP_STARTER>pl.ds.starter</INSTALLED_PACKAGES_GROUP_STARTER>
+                <INSTALLED_PACKAGES_COUNT_BULMA>1</INSTALLED_PACKAGES_COUNT_BULMA>
```

Comment:
maybe props for those 2 values will be usefull

Reply:
done